### ISSUE TITLE
Stuck at LibreLink Up

### ISSUE THREAD

**Issue by tamashi84** [timoschlueter/nightscout-librelink-up#79]
@timoschlueter 
I have a working nightscout with both heroku and azure, but cannot upload values from Libreview using your program.
When I try to use this, via heroku it is saying that the free dynos are no longer supported. I would be fine paying for the eco subscription, but I don't see a way to change the web or worker resources. Is it possible to 1.) change the resources for the subscription or 2.) integrate this as a plugin directly with Nightscout?
I am so close to making this work and I believe this will be the last step. Thanks in advance.

**Comment by virmanisukrit20** [comment]
Facing the same issue. Any update on this?

**Comment by durgok** [comment]
Same issue, even with subscription, see the same error. How did it resolve for you all? Please help.
![Capture_Heroku](https://user-images.githubusercontent.com/120686458/208996442-547a42e9-b1dc-4131-a3c7-343344866890.JPG)

**Comment by tamashi84** [comment]
I went a different direction and use Glucose Direct. You can find it here: https://github.com/creepymonster/GlucoseDirect
It bypasses the complex set up of nightscout. Enjoy!

**Comment by durgok** [comment]
Wish it has an android support, unfortunately my T1d daughter has a samsung and so it was ruled out. Back to this problem, apparently there is no solution to this problem Is it?

**Comment by virmanisukrit20** [comment]
All that we need one of the contributors to do is make a small change. The quantity needs to be 1 and the size needs to be "eco". This was told to me by someone from heroku customer service.

How do we reach out to the owners of this app? This will help my family so much with managing a child's diabetes. Also, is there a way to contribute money towards building these updates? Would be happy to do that as well

<img width="1026" alt="Screenshot 2022-12-21 at 5 08 28 PM" src="https://user-images.githubusercontent.com/100337085/209011840-66f61f14-71b0-4892-bc9b-a0c48ad7bf63.png">

**Comment by durgok** [comment]
Is it not possible for us to change in our local code set up?

**Comment by virmanisukrit20** [comment]
> Is it not possible for us to change in our local code set up?

I am not a technical person. Can you/someone walk us through with screenshots or an explanation maybe?

**Comment by virmanisukrit20** [comment]
@tamashi84 How did you solve it?

**Comment by timoschlueter** [comment]
@virmanisukrit20 thank you so much for finding this. That is actually the problem with Heroku. I am not using it myself so that I didn't look into it much. Again, thank you for investigating!
I will update the app today to resolve this problem. :)

**Comment by durgok** [comment]
Thanks, The way I worked around is by the following:
1. Fork the repository so I have its copy and edit the app.jason to replace the free with eco, commit the changes
2. Edit the readme, go to the very bottom and replace the github link of timoschlueter with my repo name so that when I hit on deploy to heroku pulls my modified app.jason from step#1 above. 
3. The deployment goes fine without errors since I subscribed to Eco plan.

This is my very first code attempt after many decades and was feeling very accomplished 😊
